pom.xml

```diff
@@ -74,7 +74,10 @@
         <dep.docker-java.version>3.3.0</dep.docker-java.version>
         <dep.jayway.version>2.6.0</dep.jayway.version>
         <dep.ratis.version>2.2.0</dep.ratis.version>
+        <dep.errorprone.version>2.18.0</dep.errorprone.version>
+        <dep.guava.version>32.1.0-jre</dep.guava.version>
```

Comment:
Where was this set before? Was this pulled in via airlift? If so, we might want to update airlift instead.

Reply:
guava version is pulled via airbase, but we see no movement on the forked airbase regarding to version upgrades.

errorprone version is introduced in this PR.

Reply:
I personally don't know why we use airbase, but I assume there's a reason. Given that we do, I think the default approach is to fix that rather introducing the new guava version here. Otherwise it's still possible we'll pull in the wrong guava version somewhere and have dependency convergence issues.

Reply:
It seems that in the past airbase was used to pull versions, but it does not appear to be the approach being used now:
- the pom also overrides other versions declared in airbase such as slf4j, jackson, joda, etc... 
- airbase v102 is in use since May 2021